shared/login/register/username-or-email/index.render.desktop.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React, {Component} from 'react'
+import {Text, Icon, Input, Button} from '../../../common-adapters'
+import Container from '../../forms/container.desktop'
+import type {Props} from './index.render'
+
+type State = {usernameOrEmail: string}
+
+class Render extends Component<void, Props, State> {
+  state: State;
```

Comment:
it complained cause i think the flow defined stuff doesn't do state: State; itself... maybe?